__Main__.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>Great! Stacking is a great approach. You should also have tried bagging. </p><p> </p><p>I like how comprehensive you have been. We are trying LOTS of permutations of models now, so this is a good search through hyperparameter space!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AI-Core-Assignments/project-1-Farhad1353/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Great thanks.</p><p>I'm a bit disappointed that at the time I did submit this project, I still had difficulty installing better packages like Xgboost, Catboost and  lightgboost.</p><p>Finally, I have managed to install them and successfully implemented them.</p><p>I am very impressed with the CatBoost.</p>